### PR TITLE
Updated Chinese Simplified translation

### PR DESCRIPTION
**Pull request by KiriKirby**
Some translated texts such as KB,MB,GB,TB,PB... have Chinese translation name. But people often use KB,MB,GB,TB,PB... So I translated as 千字节（KB）,兆字节（MB）. But those translating text may be a little long so that it may be can not full display. I hope developers can give enough space to display them. Thanks : )

### CODE REVIEW

**Comment by yaichenbaum** [comment]
@KiriKirby Can you please resolve the merge conflicts?

File: Files/MultilingualResources/Files.zh-Hans.xlf [hunk 1]
```diff
@@ -1,945 +1,944 @@
...
+          <source>Free space:</source>
+          <target state="translated">可用空间：</target>
+        </trans-unit>
+        <trans-unit id="PropertiesDriveCapacity.Text" translate="yes" xml:space="preserve">
+          <source>Capacity:</source>
+          <target state="translated">权限：</target>
+        </trans-unit>
+        <trans-unit id="PropertiesDriveFileSystem.Text" translate="yes" xml:space="preserve">
+          <source>File system:</source>
+          <target state="translated">文件系统：</target>
```
Comment (tsvietOK) [inline Files/MultilingualResources/Files.zh-Hans.xlf:939]: Do not add spaces at the end of the line if the source does not include this space. Check all your changes, please.

Comment (KiriKirby) [reply Files/MultilingualResources/Files.zh-Hans.xlf:939]: Sorry, do you mean the "space" behind the translating text? 
This is Chinese punctuation "：", it's looks like there's a space behind it.

Comment (tsvietOK) [reply Files/MultilingualResources/Files.zh-Hans.xlf:939]: Oh, didn't notice that "：" symbol is not equal to ":".

**Pull request merged**